static_src/components/panel_row.jsx

```diff
@@ -1,10 +1,7 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-
-import style from 'cloudgov-style/css/cloudgov-style.css';
-
-import createStyler from '../util/create_styler';
+import classnames from 'classnames';
```

Comment:
I think the accepted convention is to `import classNames from 'classnames';` (i.e. camelCase `classNames`).